shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -471,17 +472,29 @@ - (void)surfaceUpdated:(BOOL)appeared {
   }
 }
 
+// Count of FlutterViews currently on-screen. Used to determine when to
+// swizzle/unswizzle status bar touch event handling.
+static int visibleFlutterViewCount = 0;
```

Comment:
I'd stick tracking this count as an `NSNumber` associated object on the main application instance instead of the static here.